Deployment/Mirantis-Guidance.md

```diff
@@ -26,16 +26,24 @@ Customer is using Windows Server image 2019 with Containers
 
 ## Migration risk decision guide
 
+The possible options for the migration are combinations of the following controls:
+   - OS SKU selection
+   - container runtime selection (MCR, Moby, other or none)
+   - container runtime installation mechanism 
+   - cluster nodes update mechanism (in place, new node types or recreating the cluster)
+
 This guide is designed to help you assess the effort and risk of each migration option. 
-Criteria for successfully running MCR to host container on Azure Service Fabric cluster.
+<!-- Criteria for successfully running MCR to host container on Azure Service Fabric cluster.
```

Comment:
I have commented this out because it's not clear how is it related to the migration, or if it's a criteria for the migration or the end state.